.github/ISSUE_TEMPLATE/optimize-epic.md

```diff
@@ -1,15 +1,15 @@
 ---
-
 name: Optimize Epic
 about: This issue is used to track the progress of an Optimize Epic
-title: ''
-labels: type:epic
-assignees: ''
--------------
+title: 🎯
```

Comment:
what is this title for?

Reply:
I took it from operate/tasklist tempaltes to make it similar